tensorflow/core/kernels/cwise_ops.h

```diff
@@ -176,7 +176,11 @@ template <typename T>
 struct functor_traits<div_no_nan_op<T>> {
   enum {
     Cost = functor_traits<scalar_quotient_op<T>>::Cost + NumTraits<T>::AddCost,
+#if TENSORFLOW_USE_ROCM
```

Comment:
Would it make sense to add 'IsRocm()', 'IsCuda()' functions somewhere?
Separate PR, if feasible at all.

Reply:
Are you wanting sometthing like 'IsRocm()' to resolve at run-time or compile-time?

Reply:
Something like: 
```bool isRocm() {
#if TENSORFLOW_USE_ROCM
  return true;
#endif
  return false;
}```